### ISSUE TITLE
Sometimes spacebar opens steam community page

### ISSUE THREAD

**Issue by Helkus** [Facepunch/rust-issues#81]
No idea how to reproduce, sometimes you'll just press spacebar and it'll keep opening the steamcommunity page of the same user, have to disconnect and reconnect to make it stop

**Comment by garrynewman** [comment]
Unless this is opening a specific webpage in the overlay this is more likely to be a Steam problem. This stuff is automatic, we don't really have much control over it.

**Comment by wlfbck** [comment]
This seems to be rust specific though. I had it happening to me and a friend of mine, and it only ever happens in rust. (And we play a lot of steam games)

**Comment by fishgeekted** [comment]
A friend of mine was having similar issue. Discovered it to be the rust camera causing the Steam community screenshot  upload window to open in the background which was was waiting for keyboard input (titles and captions).

**Comment by wlfbck** [comment]
Yeah there is also another issue with alt+tabbing (and/or other forms of leaving the game window like windows-key) causing some weird behaviours of rust still capturing input (keyboard and mouse). But i'm unsure on how *exactly* to reproduce that yet.

**Comment by y0himba** [comment]
I am seeing this in game a lot.  usually happens when a person has just finished entering a chat message.  It opens the profile to the last person to enter enter a message.  I am able to close it by hitting escape a couple of times.  I use borderless window mode (-popupwindow)

**Comment by iRpiolet** [comment]
I have the same issue, it seems to happen most when I'm pressing ctrl+spacebar+W. However, that's not the only time. It also seems to open while I'm holding shift+spacebar+W. I have it open to different users, but usually it will end up being the same one, like, six or seven times in a row before it switches to another user. I don't have to quit out to get it to stop though, it just seems to pass after a while.
